static/postgres/broken-queries-ndc-metadata/schema.json

```diff
@@ -1381,7 +1381,7 @@
     "MutationsVersion": {
       "description": "Which version of the generated mutations will be included in the schema",
       "type": "string",
-      "enum": ["v1", "veryExperimentalWip"]
+      "enum": ["v1", "v2"]
```

Comment:
Something to think about - do we have (and do we plan to maintain) tests for both mutations versions?

Reply:
We have a couple of tests, probably not enough. We should probably add more.